packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -352,14 +362,14 @@ export const SignInFirstFactorMachine = SignInVerificationMachine.provide({
         return Promise.resolve(clerk.client.signIn);
       }
 
-      assertIsDefined(params);
+      assertIsDefined(params, 'First factor params');
```

Comment:
I've added a `label` to all instances of `assertIsDefined`. If `params` would be `undefined` it would say:

undefined is not defined

With the label it'll say

First factor params is not defined

👍